reservations/tests/test_recurring_reservation.py

```diff
@@ -0,0 +1,26 @@
+from assertpy import assert_that
+from django.db import connection
+from django.db.migrations.executor import MigrationExecutor
+from django.test import TestCase
+
+from reservations.models import RecurringReservation
+from reservations.tests.factories import RecurringReservationFactory
+
+
+class TestRecurringReservationUUIDMigration(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.rec = RecurringReservationFactory()
+        cls.rec_too = RecurringReservationFactory()
+
+        migration = MigrationExecutor(connection)
+        migration.migrate([("reservations", "0044_recurringreservation_description")])
```

Comment:
My understanding is that **all** migrations are run when test run is started. How running individual migrations after that works?

If my understanding is correct, when this test is being run, the database already has all these new fields, including `uuid`. I'm bamboozled! 🤔

Reply:
Yes they are run once in my understanding and that's why I'm reverting the migrations

Reply:
the main reason I have this test is that it contains now existing data rows when doing the uuid assigning